modules/openapi-generator/src/main/resources/php/model_generic.mustache

```diff
@@ -424,6 +425,7 @@ class {{classname}} {{#parentSchema}}extends {{{parent}}}{{/parentSchema}}{{^par
      * @return mixed Returns data which can be serialized by json_encode(), which is a value
      * of any type other than a resource.
      */
+    #[\ReturnTypeWillChange]
```

Comment:
You don't use `mixed` type(accepted in PHP 8) to make it compatible with PHP 7.4, am I right?

Reply:
Yes. PHP 7.4 is in the security support period, so I thought we shouldn't make breaking changes until it goes EOL.